Gothic-Unity-Project/Gothic-Unity
# Feature/load savegame - Items and player location

> Please test now, but merge only after https://github.com/Gothic-UnZENity-Project/Gothic-UnZENity/pull/39

I implemented loading of a SaveGame. We can now load original G1 save games.

**To Test**:
1. Play G1 where you removed an item and put an item to another location.
2. Save the game.
3. Open Gothic-UnZENity and seek the "Load Save Game" button on main menu
4. Select the element
5. Check if your spawning spot is correct.
6. Check if the item changes on your game are the same like when you saved it in G1
7. Please also traverse
    1. into the Old Mine and see if it loads (there was a crash before, when the world wasn't in the save game. Should work now.)
    2. Back into main world. Did you spawn where you expect to be after coming back? 


**Out-of-scope:**
* Loading of NPCs is currently disabled as there are more elements to do before we load them properly. I create another ticket for it.
